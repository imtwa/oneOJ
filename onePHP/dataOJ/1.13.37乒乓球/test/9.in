EELELEELWLEEWWE
LEWLELLWLEEWELE
LELWWWLLELLELLL
EEEEWLWLLWLWLLE
EWEWLEEELWLLLEE
ELEWLELLEEWEEWW
WWLWLLLWEEWEWLL
LEWWLELEEWLWLEL
EELWWLEWLLEELWL
LEWLEEEEWEEWWLW
LWLLELWLELWLLEL
ELWEWLWWLWLEEWW
EWWWWWELWLEELWW
LLWWLLLWLWWWWLE
ELELEEEWEWLWEWE
