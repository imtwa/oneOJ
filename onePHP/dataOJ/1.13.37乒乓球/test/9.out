0:0

0:0

0:1

0:2

0:2

1:4

1:4

3:4

3:5

4:6

5:9

5:9

6:9

6:10

6:11
0:0

3:3

3:5

3:8

3:8

3:8

3:8

5:11
2:3

2:3

3:3

4:4

4:4

4:4

5:8

5:8

5:8

5:9

6:10

6:11
0:1

0:1

1:1

1:1

7:5

7:5

8:5

9:8

11:8
0:1

0:2

0:2

2:4

2:5

2:5

4:7

5:9

5:9

6:11
0:1

1:2

1:2

1:2

1:2

2:2

2:2

6:6

7:8

8:11
0:0

0:1

1:2

5:5

5:5

7:5

11:5
1:0

2:2

2:2

11:9
0:1

0:1

0:2

0:3

0:3

0:3

1:3

3:4

4:4

0:0

0:0

0:1

0:2

0:2

1:4

1:4

3:4

3:5

4:6

5:9

5:9

6:9

6:10

6:11

9:14

9:16

9:19

9:19

9:19

9:19

11:21
2:4

2:4

3:4

4:5

4:5

4:5

5:9

5:9

5:9

5:10

6:11

6:13

6:13

7:13

7:13

13:17

13:17

14:17

15:20

17:21
0:0

0:1

0:1

2:3

2:4

2:4

4:6

5:8

5:8

6:11

7:12

7:12

7:12

7:12

8:12

8:12

12:16

13:18

14:21
0:0

0:1

1:2

5:5

5:5

7:5

12:5

13:7

13:7

21:14
1:1

1:1

1:2

1:3

1:3

1:3

2:3

4:4

5:4

